src/XliffConverter.php

```diff
@@ -24,31 +24,18 @@ final class XliffConverter
 {
     /**
      * Create a catalogue from the contents of a XLIFF file.
-     *
-     * @param string $content
-     * @param string $locale
-     * @param string $domain
-     *
-     * @return MessageCatalogue
      */
-    public static function contentToCatalogue($content, $locale, $domain)
+    public static function contentToCatalogue(string $content, string $locale, string $domain): MessageCatalogue
```

Comment:
This change is a BC break. It's not a real problem as we're about to release a new major version anyway.

Reply:
(same line 36)